doc/nrf/releases/release-notes-latest.rst

```diff
@@ -69,6 +69,12 @@ nRF9160
     This change re-introduces the optional timeout on the TCP socket used for the download.
     Upon timeout on a TCP socket, the HTTP download will fail and the ``ETIMEDOUT`` error will be returned via the callback handler.
 
+* :ref:`serial_lte_modem` application:
+
+    * Fixed a bug where an HTTP request buffer overflow could occur when FOTA was interrupted.
```

Comment:
Is that what you meant?

Reply:
Not really. There were two PRs respectively:
https://github.com/nrfconnect/sdk-nrf/pull/3503 bug fix for FOTA interrupted by AT command
https://github.com/nrfconnect/sdk-nrf/pull/3421 bug fix for HTTP request buffer overflow

Reply:
How about splitting it into two then:
```suggestion
    * Fixed a bug where FOTA downloads were interrupted when issuing an AT command.
    * Fixed an issue with overflowing HTTP request buffers. 
```